stable/mongodb/0.4.7/chart/values.yaml

```diff
@@ -0,0 +1,74 @@
+_config:
+  _metadata:
+    name: "mongodb"
+    version: "0.4.7"
+    description: ""
+  mongodb:
+    containers:
+      mongodb:
+        image: "bitnami/mongodb:3.4.2-r0"
```

Comment:
Can we make image immutable?  If we want different image, then just create different chart?